File: src/AIMS.WEB/src/app/shared/interfaces/asset.ts
export interface Asset {
  id: number;
  authorname: string; // translated
  categoryname: string; // translated
  authorid?: number; // id
  category?: number; // id
  rfidCode: string;
  assetname: string;
  issuedBy: number;
  coverimage: string;
}

File: src/AIMS.API/Domain/Models/Asset.cs
using System.Collections.Generic;
using Microsoft.AspNetCore.Http;
using System.ComponentModel.DataAnnotations.Schema;

namespace AIMS.API.Domain.Models
{
    public partial class Asset
    {
        public int Id { get; set; }
        public int? Authorid { get; set; }
        public int? Category { get; set; }
        [NotMapped]
        public IFormFile imageFile { get; set; }
        public string RfidCode { get; set; }
        public string Assetname { get; set; }
        public string Coverimage { get; set; }
    }
}
